py-scale-codec/.gitignore

```diff
@@ -0,0 +1,105 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Do we need multiple `.gitignore` anymore after removing the submodules?

Reply:
Yeah we should be able to consolidate them into a single gitignore at the root now if we want. Though I prefer do do it in a separate PR if that's ok with you. In this PR I tried to change as few things as possible so that if we encounter issues it will be easier to troubleshoot.